documentation/settings/topics/notification-channels.rst

```diff
@@ -0,0 +1,40 @@
+.. include:: /settings/media.rst
+.. _Notification-channels-doc:
+
+*********************
+Notification Channels
+*********************
+Notification Channels are API integrations with external systems to send alarm event notifications. Please provide feedback to Hyperview if you would like to see more integrations.
+
+===============
+Microsoft Teams
+===============
+To enable the integration, enter the following information.
+
+#. Give the notification channel a name. Use this name to identify the channel when you are configuring an :ref:`Alarm Policy <Alarm-policies-doc>`.
+
+#. Copy and paste the webhook URL for the target Microsoft Teams channel.
+
+|add-notification-channel|
+
+Configuring a Microsoft Teams
+-----------------------------
+Follow the following steps to configure Microsoft Teams to receive notifications from Hyperview. [#]_
+
+#. Open the channel where you want to add the webhook and select ••• from the upper-right corner.
+
+#. Select Connectors from the dropdown menu.
```

Comment:
this is not right for me, i see (triple dot) > manage channel > connectors > edit > (incoming webhook). probably a difference between new and old teams, and in-browser vs application. it would probably be better to replace this entire section with just the link to the MS documentation https://learn.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook?tabs=newteams%2Cdotnet

Reply:
Fixed. Removed instructions and point directly to MS Docs.